packages/components/molecules/f-restaurant-card/src/components/_tests/RestaurantCard.v1.test.js

```diff
@@ -236,4 +236,56 @@ describe('RestaurantCard.v1', () => {
             expect(wrapper.find('[data-test-id="restaurant-delivery-time-meta"]').exists()).toBe(true);
         });
     });
+
+    describe('Local Legend', () => {
+        it('should load an svg', () => {
```

Comment:
I'm not completely sold on the benefits of this particular test - I think the two below are likely more than adequate

Reply:
This ensures an SVG is loaded, if the icons package for example was not working, this test would fail, but also a simple manual test should catch this too, so happy to remove it

Reply:
I'm not sure if that's worth testing tbh. That would require a package update which I hope would be properly tested both manually and verified by visual tests - once we set those up. 

Not sure what would happen to the presence of the test-id if the library failed. Not sure we need to be too worried though.

Reply:
Awesome, happy to remove it